test/oli/delivery/attempts/page_lifecycle/grade_update_worker_test.exs

```diff
@@ -0,0 +1,57 @@
+defmodule Oli.Delivery.Attempts.PageLifecycle.GradeUpdateWorkerTest do
+  use Oli.DataCase
+  use Oban.Testing, repo: Oli.Repo
+
+  import Oli.Factory
+
+  alias Oli.Delivery.Attempts.PageLifecycle.GradeUpdateWorker
+
+  describe "perform/1" do
+    test "runs logic when the host is the same" do
+      section = insert(:section)
+      %{id: last_grade_update_id} = insert(:lms_grade_update, score: 100, out_of: 120)
+
+      %{id: resource_access_id} =
+        insert(:resource_access,
+          score: 5,
+          out_of: 120,
+          section: section,
+          last_grade_update_id: last_grade_update_id
+        )
+
+      {:ok, updated_resource_access} =
+        perform_job(GradeUpdateWorker, %{
+          section_id: section.id,
+          resource_access_id: resource_access_id,
+          type: :manual_batch
+        })
```

Comment:
NIT: we could add the host here

Reply:
I'm going to pass on this because we are not using the host inside the body. Please let me know if I'm missing something.

Reply:
The `host` is not being pattern-matched in the args of the `perform` function, but that does not imply that the Oban job will not have a host (since we modified the `create` function to add one)
So my idea was to explicitly pass to the `perform` function an Oban job with a host that matches the host of the test env, and assert that the Oban job is executed as expected. In this way, the test will end up mimicking better what will end up happening (receiving an Oban job with a host that matches the current one and executing the job)

Reply:
I see your point. It looks more explicit then. This PR has been paused because it looks like we are going to use another approach. But really appreciate your feedback in here. 👍